pkg/config/config.go

```diff
@@ -59,11 +59,13 @@ type IngressConfig struct {
 type MicroshiftConfig struct {
 	LogVLevel int `json:"logVLevel"`
 
-	SubjectAltNames []string      `json:"subjectAltNames"`
-	NodeName        string        `json:"nodeName"`
-	NodeIP          string        `json:"nodeIP"`
-	BaseDomain      string        `json:"baseDomain"`
-	Cluster         ClusterConfig `json:"cluster"`
+	SubjectAltNames     []string      `json:"subjectAltNames"`
+	NodeName            string        `json:"nodeName"`
+	NodeIP              string        `json:"nodeIP"`
+	KASAdvertiseAddress string        `json:"kasAdvertiseAddress"`
+	SkipKASInterface    bool          `json:"-"`
```

Comment:
What is SkipKASInterface?

At some point we should come back and add comments to all of these fields. You don't have to do all of them in this PR, but let's at least add good docs for the new things.